static/app/utils/profiling/profile/chromeTraceProfile.tsx

```diff
@@ -138,15 +138,20 @@ function buildProfile(
   }
 
   const firstTimestamp = beginQueue[beginQueue.length - 1].ts;
+  const lastTimestamp = endQueue[0]?.ts ?? beginQueue[0].ts;
```

Comment:
We should probably just check for length above this. If begin queue is empty there is nothing to do anyways and we could just return. I think even firstTimestamp will crash right now if there are 0 events (tho we’ll likely never end up there because of how we check for chrometrace format in the guard), but it would be a tad cleaner

Reply:
There's already a length check on the `beginQueue` so that shouldn't be a problem.